commit-message.md

```diff
@@ -0,0 +1,26 @@
+# Guidelines to create great commits messages
+
+
+We use [GitCop](https://gitcop.com) to check that commit messages are
+properly written. The rules are the following:
+
+* The first line of a commit message, called the subject line should
+  not be more than 80 characters long.
+
+* The commit message should end with the following trailers:
+
+  Licence: MIT
+  Signed-off-by: User Name <email@address>
+
+  where "User Name" is the author's real name and email@address one of
+  his valid email addresses.
+
+  These trailers means that the author agrees with the following
+  document:
+
+  developer-certificate-of-origin
```

Comment:
I'd just inline a link here:

    …agrees with the [Developer Certificate of Origin][DCO].

And then define the reference at the end of this file:

    [DCO]: ./developer-certificate-of-origin

Similarly with the link to `LICENSE` below.